tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -134,6 +134,10 @@ function main() {
   source tools/python_bin_path.sh
 
   pushd ${TMPDIR}
+
+  # No Python unit test files should go into the PIP package.
+  rm -f $(find ./ -name '*_test.py')
```

Comment:
I would much rather we adjust the dependencies of the build_pip_package rule instead. We already have too many point at which we control what goes into that package.